src/pypi_changes/_cli.py

```diff
@@ -43,6 +43,14 @@ def _define_cli_arguments() -> ArgumentParser:
     cache_help = "seconds how long requests should be cached (pass 0 to bypass the cache, -1 to cache forever)"
     parser.add_argument("--cache-duration", "-d", default=3600, type=int, help=cache_help, metavar="SEC")
 
+    parser.add_argument(
+        "--alphabetize",
+        "-a",
+        help="sort output alphabetically",
+        action="store_true",
+        dest="alphabetize",
+    )
```

Comment:
Can we make this a mutually exclusive flag with `--updated` and make this `--alphabetic`? And make the former the default; in the spirit of explicit is better than implicit.